test/unit/core/selectors.ts

```diff
@@ -278,6 +311,14 @@ suite('selectors', ({ expect, stub }) => {
     });
   });
 
+  describe('productsLoaded()', () => {
+    it('should return if products are loaded', () => {
```

Comment:
`should return productsLoaded value` or some kind of rewording. The above wording makes it sound like it will return in the case that products are loaded.